schwifty/bank_registry/manual_lu.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "country_code": "NL",
```

Comment:
Shouldn't this be LU?

Reply:
correct, fixed!

Reply:
but I'm guessing this also requires the bank code to be numeric as stated [here](https://wise.com/gb/iban/luxembourg)
I'll try and find it